src/editor.js

```diff
@@ -916,7 +917,7 @@ function showInfo(data) {
     }
 
     // Display the project name
```

Comment:
Again, had issues when testing (race condition where projectData wasn't populated?) realized when trying to debug it that this should be testing the more local variable - has no effect on application, but left it because it's how it should be.